### PR TITLE
feat: add yoctopuce pip to python.yaml

### PR DESCRIPTION
**Pull request by nikosmitrop1995**
## Package name:

`yoctopuce-pip`

## Package Upstream Source:

[yoctolib_api](https://github.com/yoctopuce/yoctolib_python)

## Purpose of using this:

This dependency is being used for controlling the [Yoctopuce](https://www.yoctopuce.com/) devices using their Python API. This makes easier to use and control actuators and sensors.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - [https://pypi.org/project/yoctopuce/](https://pypi.org/project/yoctopuce/)
- Ubuntu: https://packages.ubuntu.com/
  - [https://pypi.org/project/yoctopuce/](https://pypi.org/project/yoctopuce/)
- Fedora: https://packages.fedoraproject.org/
  - Not available
- Arch: https://www.archlinux.org/packages/
  - Not available
- Gentoo: https://packages.gentoo.org/
  - Not available
- macOS: https://formulae.brew.sh/
  - Not available
- Alpine: https://pkgs.alpinelinux.org/packages
  - Not available
- NixOS/nixpkgs: https://search.nixos.org/packages
  - Not available
- openSUSE: https://software.opensuse.org/package/
  - Not available
- rhel: https://rhel.pkgs.org/
  - Not available

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -9904,6 +9904,10 @@ yapf3:
   nixos: [python3Packages.yapf]
   ubuntu:
     '*': [yapf3]
+yoctopuce-pip:
```
Comment (clalancette) [inline rosdep/python.yaml:9907]: Our current best practice is to put `python3-` in front of the key:

```suggestion
python3-yoctopuce-pip:
```

(don't forget to re-alphabetize this once you do that)

**Review by clalancette — DISMISSED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (nikosmitrop1995) [reply rosdep/python.yaml]: I have renamed it to `python3-yoctopuce` and placed it to the correct order alphabetically. Do you think it's better to add it at `base.yaml` ?

Comment (clalancette) [reply rosdep/python.yaml]: > I have renamed it to `python3-yoctopuce` and placed it to the correct order alphabetically. Do you think it's better to add it at `base.yaml` ?

No, it should be in `python.yaml`, but it should be called `python3-yocotopuce-pip`.

Comment (nikosmitrop1995) [reply rosdep/python.yaml]: Done

**Review by quarkytale — APPROVED** [review]

**Review by audrow — APPROVED** [review]

**Pull request merged**